assets/component-card.css

```diff
@@ -12,12 +12,32 @@
 
 .card--card,
 .card--standard .card__inner { 
-  background-clip: padding-box;
   border-radius: var(--card-corner-radius);
   border: var(--card-border-width) solid rgba(var(--color-foreground), var(--card-border-opacity));
+  position: relative;
+}
+
+.card--card:after,
+.card--standard .card__inner:after {
+  content: '';
+  position: absolute;
+  width: calc(var(--card-border-width) * 2 + 100%);
+  height: calc(var(--card-border-width) * 2 + 100%);
+  top: calc(var(--card-border-width) * -1);
+  left: calc(var(--card-border-width) * -1);
+  z-index: -1;
+  border-radius: var(--card-corner-radius);
   box-shadow: var(--card-shadow-horizontal-offset) var(--card-shadow-vertical-offset) var(--card-shadow-blur-radius) rgba(var(--color-foreground), var(--card-shadow-opacity));
+}
+
+.card .card__inner .card__media {
   overflow: hidden;
-  position: relative;
+  border-radius: calc(var(--card-corner-radius) - var(--card-border-width) - var(--card-image-padding));
```

Comment:
It's because the border radius of the card in your screenshot is too small. If you increase it (or reduce the card padding, you'll see the radius of the image become apparent. The "inner" border radius needs to be reduced proportionally by the offset distance which at a certain point just ends up making the radius 0. If we didn't reduce the inner radius like this you'd end up with this effect https://screenshot.click/13-24-pmp9d-k9g0y.png

Reply:
I actually recorded this for another PR that demonstrates the same thing https://screenshot.click/10-01-1ly6v-t317y.mp4

Reply:
> we're talking about the image getting border radius applied as well?

Yes!  The only trade off that I am observing is that if merchants increase image padding with a subtle radius, [they lose that styling](https://screenshot.click/21-12-toxng-z8hwb.png) and have to discover the relationship of the application of the roundness by  testing themselves. This is an edge case to me and not a concern to ship for now but worth  sharing with  @danielvan and  @nicklepine